### DISCUSSION TITLE
docker build时报错，请问，有没有快速的解决办法？

### DISCUSSION THREAD

**Discussion by wf-1202** [lobehub/lobehub#2251] [Self Hosting] [answered]
二月底三月初左右，封装了一次docker成功。
最近服务器重启了，需要重新部署一次。
部署时报错，如下：
./src/app/chat/features/ChatHeader/ShareButton/style.ts:36:21
Type error: Property 'markdownInChat' does not exist on type 'FullStylish'.

  34 |     border-bottom: 1px solid ${token.colorBorder};
  35 |   `,
> 36 |   markdown: stylish.markdownInChat,
     |                     ^
  37 |   preview: cx(
  38 |     stylish.noScrollbar,
  39 |     css`
 ELIFECYCLE  Command failed with exit code 1.
 
 和最新版本项目中代码相比，发现，最新的项目代码中，这部分内容已经没有了。
 感觉是代码重构，或 初始化依赖时，依赖中的代码有变动。
 请问，有没有什么快速的解决办法？
 还是说，只能在最新的项目代码中，重新构建部署？因为代码中有改动，算是略微二次开发吧，不想重写一遍~

**Comment by arvinxx** [comment, marked as answer]
可以基于最新的代码开个分支，把你的二开部分的内容cherry pick过去。不一致的地方就得自行解决了。
